build-tools/xabootstrap/xabootstrap/NOTICE.txt

```diff
@@ -0,0 +1,2 @@
+﻿http://www.kajabity.com

```

Comment:
What's this file for?

Reply:
There *should* be a `README.md` in this directory with required documentation...

Reply:
Accidentally added, probably coming from the nuget